share/revng/test/configuration/revng-qa/for-data-layout-analysis.yml

```diff
@@ -0,0 +1,21 @@
+tags:
+  - name: for-data-layout-analysis
+    variables:
+      CFLAGS:
+        - -O2
+        - -Wno-pointer-bool-conversion
+        - -fno-unroll-loops
+        - -fno-inline
+        - -fno-optimize-sibling-calls
+        - -fno-stack-protector
+sources:
+  - tags: [for-data-layout-analysis]
+    members:
+      - share/revng/test/tests/analysis/data-layout-analysis/linked-lists.c
+      - share/revng/test/tests/analysis/data-layout-analysis/segments-and-sections.c
```

Comment:
Use `prefix: share/revng/test/tests/analysis/data-layout-analysis/`. See `for-detect-abi.yml` as an example, in `revng`.